app/uk/gov/hmrc/excisemovementcontrolsystemapi/services/AuditService.scala

```diff
@@ -103,4 +103,14 @@ class AuditService @Inject() (auditConnector: AuditConnector, appConfig: AppConf
       val event = AuditEventFactory.createGetMovementsDetails(request, response, userDetails, authExciseNumber)
       auditConnector.sendExplicitAudit("GetInformation", event)
     }
+
+  def getInformation(
```

Comment:
Can we avoid this being an overloaded method? Would prefer it to be named differently so it's really clear what is being used where

Reply:
I was expecting you might not be a fan of the overloaded method. Could rename the methods after the requestType (auditService.getMovements & auditService.getSpecificMovmement) or append for auditService.getInformationForSpecificMovement (but that's so long I'd almost want to proxy it)